lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/sys.rb

```diff
@@ -1019,7 +1019,9 @@ def cmd_reg(*args)
               print_error('Invalid characters provided! Could not fully convert data provided to -d argument!')
               return false
             end
-           data = data.pack("C*")
+            data = data.pack("C*")
```

Comment:
Similar to the above validation that's been added for REG_BINARY hex values, should we verify dword/qword/etc are digits only `\d+`?

A user might assume consistency between REG_BINARY and setting hex values for qword/dword, but the end result is writing 0 to the registry and reporting a successful write to the user

Reply:
That's a good call, I'll make that change!